lib/plugins/aws/package/compile/events/eventBridge/index.test.js

```diff
@@ -1334,3 +1336,121 @@ describe('AwsCompileEventBridgeEvents', () => {
+                  eventBridge: {
+                    eventBus: 'arn:aws:events:us-east-1:12345:event-bus/default',
+                    schedule: 'rate(10 minutes)',
+                  },
+                },
+              ],
+            },
+          },
+        })
+        .then(fixturePath =>
+          runServerless({
+            cwd: fixturePath,
+            cliArgs: ['package'],
+          })
+        )
+        .then(({ cfTemplate, awsNaming }) => {
+          ({ Resources: cfResources } = cfTemplate);
+          naming = awsNaming;
+        })
+    );
+
+    it('should create the correct execution role', () => {
+      const executionRolePolicies =
+        cfResources.IamRoleCustomResourcesLambdaExecution.Properties.Policies;
+      expect(executionRolePolicies[0].PolicyDocument.Statement).to.deep.equal([
+        {
+          Action: [
+            'events:PutRule',
+            'events:RemoveTargets',
+            'events:PutTargets',
+            'events:DeleteRule',
+          ],
+          Effect: 'Allow',
+          Resource: {
+            'Fn::Join': [
+              ':',
+              [
+                'arn',
+                { Ref: 'AWS::Partition' },
+                'events',
+                {
+                  Ref: 'AWS::Region',
+                },
+                {
+                  Ref: 'AWS::AccountId',
+                },
+                'rule/service-dev-foo-rule-1',
+              ],
+            ],
+          },
+        },
+        {
+          Action: ['lambda:AddPermission', 'lambda:RemovePermission'],
+          Effect: 'Allow',
+          Resource: {
+            'Fn::Join': [
+              ':',
+              [
+                'arn',
+                { Ref: 'AWS::Partition' },
+                'lambda',
+                {
+                  Ref: 'AWS::Region',
+                },
+                {
+                  Ref: 'AWS::AccountId',
+                },
+                'function',
+                'service-dev-foo',
+              ],
+            ],
+          },
+        },
+      ]);
+    });
+
+    it('should create the necessary resources', () => {
+      expect(cfResources.FooCustomEventBridge1).to.deep.equal({
+        Type: 'Custom::EventBridge',
+        Version: 1,
+        DependsOn: [
+          'FooLambdaFunction',
+          naming.getCustomResourceEventBridgeHandlerFunctionLogicalId(),
+        ],
+        Properties: {
+          ServiceToken: {
+            'Fn::GetAtt': [naming.getCustomResourceEventBridgeHandlerFunctionLogicalId(), 'Arn'],
+          },
+          FunctionName: 'service-dev-foo',
+          EventBridgeConfig: {
+            EventBus: 'arn:aws:events:us-east-1:12345:event-bus/default',
+            Input: undefined,
+            InputPath: undefined,
+            InputTransformer: undefined,
+            RuleName: 'service-dev-foo-rule-1',
+            Pattern: undefined,
+            Schedule: 'rate(10 minutes)',
+          },
+        },
+      });
```

Comment:
Same here we should use `naming` for logical id resolution, and confirm just on few core properties

Reply:
I have made some changes, but I wasn't sure which names I need to extract from the naming object. Let me know if I should add any of the other ones.